.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
```

Comment:
improve what?

I would rewrite this entire sentence anyway, it has no relation to reporting bugs.